astropy/utils/iers/data/Leap_Second.dat

```diff
@@ -1,10 +1,10 @@
 #  Value of TAI-UTC in second valid beetween the initial value until
 #  the epoch given on the next line. The last line reads that NO
 #  leap second was introduced since the corresponding date
-#  Updated through IERS Bulletin 58 issued in July 2019
+#  Updated through IERS Bulletin 59 issued in January 2020
 #
 #
-#  File expires on 28 June 2020
+#  File expires on 28 December 2020
```

Comment:
Does this need a change log?

Reply:
Hmm, so far we have not done that for IERS_B at least. I think it is something a user should be able to assume has happened....